bench/many_deep.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+
+<head></head>
+
+<body>
+  <script type="module">
+    import { hydrateShadowRoots as manualWalkHydration } from '../_implementation/manual_walk.js';
+    import { hydrateShadowRoots as querySelectorHydration } from '../_implementation/queryselectorall.js';
+
+    function benchmark(hydrateMethod) {
+      let hydrateShadowRoots;
+      if (hydrateMethod === 'manualWalk') {
```

Comment:
Can this same benchmark be run against native?

Reply:
In theory, yes. I had a bit of trouble getting Chrome 82 working with tachometer. WIP.